tests/unit/worker/test_remote_submission_worker.py

```diff
@@ -0,0 +1,47 @@
+import mock
+import os
+import responses
+import shutil
+import unittest
+
+import scripts.workers.remote_submission_worker as remote_submission_worker
+
+
+class TestDownloadAndExtractFile(unittest.TestCase):
+    def setUp(self):
+        self.download_dir = "/tmp/evalai/"
+        self.download_filename = "file"
+        self.download_path = "{}{}".format(self.download_dir, self.download_filename)
+
+        self.url = "http://example.com"
+        self.body = """{"content": "test-data"}"""
+        responses.add(
+            responses.GET,
+            self.url,
+            body=self.body,
+            status=200
+        )
+
+        remote_submission_worker.create_dir(self.download_dir)
+
+    def tearDown(self):
+        shutil.rmtree(self.download_dir)
+
+    @responses.activate
+    def test_download_and_extract_file_with_correct_url(self):
+        self.assertTrue(os.path.exists(self.download_dir))
```

Comment:
Here the testing code checks if the directory is created successfully, and if not, the test fails with the message indicating that the directory creation went wrong.